### ISSUE TITLE
[Bug] events not returning data

### ISSUE THREAD

**Issue by oeyvindb** [OpenBB-finance/OpenBBTerminal#2638]
**Describe the bug**
The events option under economy is to returning data and is complaining that it has no end date even when I specify end date in the terminal

**To Reproduce**
economy -> events

<img width="591" alt="Skjermbilde 2022-09-26 kl  21 48 27" src="https://user-images.githubusercontent.com/10116900/192367177-15a3d815-069a-4276-af0b-6f0839b7694f.png">

<img width="483" alt="Skjermbilde 2022-09-26 kl  21 48 05" src="https://user-images.githubusercontent.com/10116900/192367202-213d7507-6de7-4ccf-8787-0f7cd6834372.png">

**Desktop (please complete the following information):**

- OS: OSX
- Python version 3.9.14

**Comment by jmaslek** [comment]
Duplicate of #2529 

I believe this is waiting on an installer update @montezdesousa ?

**Comment by montezdesousa** [comment]
> Duplicate of #2529 
> 
> I believe this is waiting on an installer update @montezdesousa ?

Correct

**Comment by colin99d** [comment]
Since this is a duplicated of a closed issue I will close it.
